docs/deployed-contracts.md

```diff
@@ -97,3 +97,12 @@
 - Curve Liquidity Farming:
 
   - Pool Contract: [`0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E`](https://goerli.etherscan.io/address/0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E)
+
+## Protocol Mocks
+
+### Rinkeby
```

Comment:
Do we have Oracle here?

Reply:
We don't have a standalone contract for oracle in protocol mock, EOA address can simulate its reports.